### PR TITLE
feature: offline table index

### PR DESCRIPTION
**Pull request by wfxr**
Template for table schema becomes very complicated when supporting optional primary key and index. So I rewrite it using normal functions.

### CODE REVIEW

File: internal/database/dbutil/schema.go [hunk 1]
```diff
@@ -58,61 +23,96 @@ func (c ColumnList) Names() []string {
...
+		types.BackendSQLite:
+		return fmt.Sprintf("CREATE INDEX %s ON %s (%s)", qt(indexName), qt(tableName), qt(fields...))
+	case types.BackendSnowflake:
+		return fmt.Sprintf("ALTER TABLE %s CLUSTER BY (%s)", qt(tableName), qt(fields...))
+	default:
+		panic(fmt.Sprintf("unsupported backend type %s", backend))
+	}
+}
+
+func parseColumns(entity *types.Entity, withUnixMillis bool, features types.FeatureList, backend types.BackendType) (rs []Column) {
```
Comment (YikSanChan) [inline internal/database/dbutil/schema.go:78]: withUnixMillis -> isCDC?

**Review by jinghancc — APPROVED** [review]
LGTM

**Review by YikSanChan — APPROVED** [review]
lgtm

File: internal/database/dbutil/schema.go (hunk 1 above)
Comment (wfxr) [reply internal/database/dbutil/schema.go]: fixed

**Pull request merged**